src/realm/object-store/impl/object_accessor_impl.hpp

```diff
@@ -135,13 +135,7 @@ class CppContext {
     }
     util::Any box(object_store::Dictionary v) const
     {
-        AnyDict ret;
-        for (const auto& it : v) {
-            std::string key{it.first.get_string()};
-            auto value = box(it.second);
-            ret.emplace(key, value);
-        }
-        return ret;
+        return v;
```

Comment:
The previous implementation here was incorrect; box() is supposed to box the value, not copy it into a different data structure. This does not appear to have been used anywhere.